addons/sitting/XEH_clientInit.sqf

```diff
@@ -3,11 +3,12 @@
 // Exit on Headless
 if (!hasInterface) exitWith {};
 
+GVAR(isEnabled) = false;
 ["SettingsInitialized", {
-    TRACE_1("SettingInit", GVAR(enable));
-
+    if ([[QUOTE(ADDON), QGVAR(enable)], ["acex_sitting", "acex_sitting_enable"], "3.8.0"] call EFUNC(common,deprecateComponent)) exitwith {};
     //If not enabled, then do not add CanInteractWith Condition or event handlers:
-    if (!GVAR(enable)) exitWith {};
+    if (!GVAR(enable) || GVAR(isEnabled)) exitWith {};
+    GVAR(isEnabled) = true;
```

Comment:
How could it? Unless someone put 2 of those down (in which case he should get all the RPT warnings).